admin-frontend/src/pages/Status.js

```diff
@@ -175,21 +224,17 @@ class Status extends React.Component {
       };
 
       return (
-        <div
-          className={`badge-${mappings[status] || "success"}`}
-          style={{
-            width: 10,
-            height: 10,
-            borderRadius: 20,
-            marginLeft: 5,
-            display: "inline-block",
-          }}
-        />
+        <>
+        <StatusBadge status={mappings[status] || "success"}>
+
+        </StatusBadge>
+        </>
       );
     };
 
     return (
-      <div>
+      <>
```

Comment:
?

Reply:
Whats is wrong with this? I don't need to use a <div> with no classes to return more than one element, I can use <> </> to do this.